xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -642,15 +642,6 @@ void CGUIWindowVideoNav::LoadVideoInfo(CFileItemList &items, CVideoDatabase &dat
       if (!pItem->m_bIsFolder && !fetchedPlayCounts)
       {
         database.GetPlayCounts(items.GetPath(), items);
-        // for addons if playcount is still -1 set it to zero
```

Comment:
yes it is not needed any more as the getPlayCount if playcount is not set returns zero which was the outcome needed.